src/test/store/markers.test.js

```diff
@@ -123,6 +124,86 @@ describe('selectors/getMarkerChartTimingAndBuckets', function() {
       ]);
     });
   });
+
+  describe('network markers', function() {
+    function getTimingAndBucketsForNetworkMarkers(testMarkers) {
+      const profile = getProfileWithMarkers(testMarkers);
+
+      // Let's monkey patch the returned profile to set the Network category on
+      // all markers.
+      const networkCategory = ensureExists(profile.meta.categories).findIndex(
+        category => category.name === 'Network'
+      );
+      profile.threads.forEach(thread => {
+        thread.markers.category.fill(networkCategory);
+      });
```

Comment:
This is a bit messy; this is usually not needed in other tests, but here it's needed because of the check for the bucket `Network` in the sorting function.

Ideally the tool to create markers should support categories but I was a bit lazy as this is used everywhere...

Reply:
heh, it's fine by me. It's only in the tests after all :)